### DISCUSSION TITLE
Make certain calendars transparent?

### DISCUSSION THREAD

**Discussion by guitarbeat** [coedit/gcal-rainbow-surf#5] [archive]
Some of my calendars are viewed as less real, and i'd love to make them transparent but still there if I knew out. Also, a menu setting place would be perfect for this extension.

**Comment by ezdub** [comment]
Hi @guitarbeat, thank you for starting this conversation for this interesting idea. I can see where this would look great, for example I'd love to tone down the holiday calendars. As you mention, a settings menu would be needed to configure this, which doesn't exist yet. 
Anyone is welcome to code this and submit a PR.

**Anyone who would like this feature added, please thumbs-up this post to help prioritize it.**
